spec/vehicle_spec.rb

```diff
@@ -33,4 +34,21 @@
       expect(@camaro.electric_vehicle?).to eq(false)
     end
   end
+
+  describe '#set_date' do
```

Comment:
Yes, good job testing your helper method. You'd also want to test your other `set_plate` helper method.